corehq/apps/cloudcare/templates/preview_app/too_many_ucrs_preview.html

```diff
@@ -0,0 +1,10 @@
+{% extends "formplayer-common/base.html" %}
+{% load hq_shared_tags %}
+{% load compress %}
+{% load i18n %}
+
+{% block body %}
+<div>
```

Comment:
not necessary to wrap a `<p>` in an unstyled `<div>`

I would recommend styling the `div` with the css class `alert alert-warning`

Reply:
![Screenshot from 2023-10-31 10-34-35](https://github.com/dimagi/commcare-hq/assets/15785053/13624d87-0214-478e-b4c7-04b4f7ad03fe)
Updated!

Reply:
a1f41126477e116b471718cc09c8149a07f107d6